src/migrations/20220603081324-add-archive-at-to-feature-toggle.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+exports.up = function (db, callback) {
+    db.runSql('ALTER TABLE features ADD "archived_at" date;', callback);
```

Comment:
Should we drop the `archived` column, and only use `archived_at`?

Reply:
that could lead us to breaking our contract, we must have the archived flag in the response, and that would require more mapping.

Reply:
Yeah. Would be nice with one db column, and instead add `archived` to the response in the handler.